ingestion_server/README.md

```diff
@@ -116,3 +116,11 @@ database_table_to_elasticsearch_model = {
     'image': Image
 }
 ```
+
+# Deployment (last deployed version: 1.20.0)
+
+This codebase is deployed as Docker image to Docker hub. The deployed image is then pulled in the production environment. See the [`./publish_release.sh`](publish_release.sh) script for deploying to Docker hub. The machine running the script must be logged into Docker as the Openverse account.
+
+The current docker hub image is at [openverse/ingestion_server](https://hub.docker.com/r/openverse/ingestion_server) but the image previously lived at [creativecommons/ingestion_server](https://hub.docker.com/r/creativecommons/ingestion_server).
+
+The version numbering for the Docker image appears to be a standalone semver value, seperate from the repository version.
```

Comment:
A little typo here: "... _separate_ from the repository version"

Reply:
```suggestion
The version numbering for the Docker image appears to be a standalone semver value, separate from the repository version.
```